fishcore/About.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace fishcore
+{
+    public class About
```

Comment:
New class for testing how to use Fetch. The "About" class contains a single string for passing back to the front end.